src/components/EndpointExplorer.js

```diff
@@ -102,7 +102,18 @@ function buildRequestUrl (baseUrl, endpoint, values) {
   _.each(template.varNames, (varName) => {
     let objectPath = (varName in endpoint.path) ?
       endpoint.path[varName] : varName + '.value';
-    let value = _.get(values, objectPath);
+    let value;
+
+    if (typeof objectPath === 'string') {
+      value = _.get(values, objectPath);
+    } else if (typeof objectPath === 'function') {
+      try {
+        value = objectPath(values);
+      } catch (e) {
+        console.log(`Extraction function in ${endpoint.label} path.${varName} failed`)
```

Comment:
Now looking back, this was a kludge to prevent having to check for `undefined` in the (now named) creditPrepender. The better solution was to solve the copypasta as you had recommended in another comment. Thanks!